test/hotspot/jtreg/ProblemList-Xcomp.txt

```diff
@@ -27,23 +27,15 @@
 #
 #############################################################################
 
-vmTestbase/nsk/jvmti/SetFieldAccessWatch/setfldw001/TestDescription.java 8205957 generic-all
+serviceability/dcmd/vm/ClassLoaderStatsTest.java 8278123 generic-all
 
-vmTestbase/vm/mlvm/mixed/stress/regression/b6969574/INDIFY_Test.java 8265295 linux-x64,windows-x64
+vmTestbase/nsk/jvmti/SetFieldAccessWatch/setfldw001/TestDescription.java 8205957 generic-all
 
-vmTestbase/nsk/jvmti/scenarios/sampling/SP07/sp07t002/TestDescription.java 8245680 windows-x64
+serviceability/jvmti/vthread/ContFramePopTest/ContFramePopTest.java 8278053 generic-all
+serviceability/jvmti/vthread/ContStackDepthTest/ContStackDepthTest.java 8278053 generic-all
 
 serviceability/sa/TestJhsdbJstackMixed.java 8248675 linux-aarch64
 
-compiler/c2/irTests/TestSkeletonPredicates.java 8286361 generic-all
```

Comment:
It might be a merge problem. My PR was submitted before 8286442. I remereged changes, however my merge commit also doesn't have removal of this line:
https://github.com/openjdk/jdk/pull/8604/commits/8255e760ff1a2ff616f00f559de29e66d7036b39

So I am really confused and can't understand how I managed to introduce this problem.

Reply:
That is really bizarre. No individual commit shows the line being deleted, yet in the final commit it is.